observability/network_observability/network-observability-secondary-networks.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="network-observability-secondary-networks"]
+= Secondary networks
+include::_attributes/common-attributes.adoc[]
+:context: secondary_networks
+
+toc::[]
+You can configure the Network Observability Operator to collect and enrich network flow data from secondary networks, such as SR-IOV and Open Virtual Network (OVN)-Kubernetes.
+
+// Note to tech review:
+// Is the existing SR-IOV example we have, "Configuring monitoring for SR-IOV interface traffic", an example of secondary network? If so, it is not through a VM, right? 
+
+.Prerequisites
+* Access to an {product-title} cluster with an additional network interface, such as a secondary interface or an L2 network.
+* The `spec.agent.ebpf.privileged` field must be set to `true`.
```

Comment:
This shouldn't be considered a prerequisite since it's providing the instructions to do this in the two scenarios below.